.storybook/stories/KvCauseSelector.stories.js

```diff
@@ -0,0 +1,125 @@
+import KvCauseSelector from '@/components/Kv/KvCauseSelector';
+
+export const causeList = [
```

Comment:
If you don't export this, it won't be picked up as a story and you don't have to exclude it below. It should just be a global variable in this file.

Reply:
Oh, I'm using it in a WIP story for KvCarousel right now

Reply:
I see, if you're going to be using it in multiple stories, then maybe lets add it as a separate file in `mock-data`